src/Infrastructure/Services/UserService.cs

```diff
@@ -916,8 +916,7 @@ public async Task<Response<SignUpResponse>> SignUpAsync(string email, string pas
                         await _emailService.SendEmailAsync(email, "Be Part of Research registration attempt", htmlBody);
                     }
 
-                    return Response<SignUpResponse>.CreateSuccessfulContentResponse(
-                        new SignUpResponse { IsSuccess = true, }, _headerService.GetConversationId());
+                    throw new UsernameExistsException("User already exists");
```

Comment:
Make the messages consistent. As this is already a `UsernameExistsException` just passe the email address as the message.